homework-04/routes/products.js

```diff
@@ -50,9 +40,10 @@ router.get('/:id', (req, res) => {
 
 router.post('/', (req, res) => {
   if (req.body !== undefined) {
-    const newProduct = new Product(uuidv1(), req.body.name)
-    productDb.push(newProduct)
-    res.status(201).json(newProduct)
+    Product.create({name: req.body.name})
```

Comment:
[OPTIONAL]: add some way to post a review. You can do it, for example, by creating `post('/product/:id/reviews')` route

Reply:
Done